### DISCUSSION TITLE
Next js Api end points are not returning data saved in database after build has done

### DISCUSSION THREAD

**Discussion by BenoitIT** [vercel/next.js#66281] [Help]
### Summary

I deployed next js app on vercel which binds both backend and frontend together ,prisma was used as ORM and postgres database is used. the problem is , i am saving data to the database but data can not be returned by the endpoints unless I rebuild the app.

**Comment by icyJoseph** [comment]
Hi,

Most likely, your GET endpoint got `statically optimized`, this default behavior will change in Next 15 IIRC, in the mean time you can follow these to opt out of it:

- https://nextjs.org/docs/app/building-your-application/routing/route-handlers#opting-out-of-caching

**Comment by BenoitIT** [comment]
thank you Joseph!
